core/roboconf-target-embedded/src/main/java/net/roboconf/target/embedded/internal/EmbeddedHandler.java

```diff
@@ -51,26 +62,46 @@ public String getTargetId() {
 
 	@Override
 	public String createMachine( TargetHandlerParameters parameters ) throws TargetException {
-
+		if(ipTable == null && parameters != null && parameters.getTargetProperties() != null) {
+			String ips = parameters.getTargetProperties().get(IP_ADDRESSES);
+			if(ips != null) {
+				String iplist[] = ips.trim().split("\\s*,\\s*");
+				for(String ip : iplist) {
+					if(ipTable == null) ipTable = new HashMap<>();
```

Comment:
This line would be useless if iPtable had been initialized in the class with `new HashMap<>( 0 )`.